packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -126,19 +127,25 @@ export class AccordionItem extends LitElement {
     }, options);
   }
 
-  // #endregion
+  //#endregion
 
-  // #region Events
+  //#region Events
+
+  /** Fires when the component's content area is collapsed. */
+  calciteAccordionItemCollapsed = createEvent({ cancelable: false });
+
+  /** Fires when the component's content area is expanded. */
+  calciteAccordionItemExpanded = createEvent({ cancelable: false });
```

Comment:
fixme: Why are the event names in past tense? Shouldn't they be `calciteAccordionItemCollapse` and `calciteAccordionItemExpand`? They are inconsistent with `open` and `close` events.